### DISCUSSION TITLE
Memperbaiki label di atas kolom

### DISCUSSION THREAD

**Discussion by ToniAS21** [teamalgoritma/community#220] [Q&A] [answered]
Bagaimana mengatur label di atas kolom agar rapi ?
![Screenshot (1698)](https://user-images.githubusercontent.com/83503249/180391422-cf08c0d7-a89c-4478-84e4-85dbbe68ca5e.png)

Sekian terima kasih.

**Comment by dwiknrd** [comment, marked as answer]
Dear [ToniAS21](https://github.com/ToniAS21),
Untuk masalah tersebut anda dapat mengatur beberapa parameter pada fungsi `geom_label()`, yaitu:
- `position = position_dodge(width=...)` -> untuk  mengatur posisi jarak antar label
- `vjust = 1` -> untuk mengatur posisi label secara horizontal

Semoga bermanfaat :)

**Reply by ToniAS21** [reply]
Baik terima kasih sudah bisa.
